src/Uno.UWP/Storage/StorageFile.Base.cs

```diff
@@ -95,11 +102,35 @@ public virtual async Task MoveAndReplaceAsync(CancellationToken ct, IStorageFile
 					await src.CopyToAsync(dst, Buffer.DefaultCapacity, ct);
 				}
 
+				await CopyExifOrientationAsync(Owner, target);
+
 				await DeleteAsync(ct, StorageDeleteOption.PermanentDelete);
 
 				Path = target.Path;
 			}
 
+
+			private static
+#if __ANDROID__
+			async
+#endif
+			Task CopyExifOrientationAsync(StorageFile owner, IStorageFile target)
+			{
+#if __ANDROID__
+				if (owner.ContentType == "image/jpeg" &&
+					target is StorageFile { Implementation: Local impl })
```

Comment:
Should we also test for the source file origin?

Reply:
@jeromelaban Intent was that source's stream can always be opened, but I was concerned about the "save" part as I was unsure whether it will work properly for Saf implementation or not.

Anyway, I just realized that the change in `ImageSource` is sufficient, and we don't need to do anything with copying.

For some reason, when I was testing, `GetAttribute` on destination was returning normal orientation even if the source is not. But it's just something weird with the ExifInterface API and the exif information was actually not lost. It's just ImageSource need to respect it.